job/validators.py

```diff
@@ -402,6 +402,16 @@ def update_pv_defaults_offgrid(self):
                 if self.models["ElectricUtility"].outage_end_time_step > max_ts:
                     self.add_validation_error("ElectricUtility", "outage_end_time_step",
                                               f"Value is greater than the max allowable ({max_ts})")
+            if self.models["ElectricUtility"].outage_start_time_steps:
+                max_start_time_step_input = max(self.models["ElectricUtility"].outage_start_time_steps)
+                if max_start_time_step_input > max_ts:
+                    self.add_validation_error("ElectricUtility", "outage_start_time_steps",
+                                                f"Value is greater than the max allowable ({max_ts})")
+
+                if self.models["ElectricUtility"].outage_durations:
+                    if max_start_time_step_input + max(self.models["ElectricUtility"].outage_durations) > max_ts:
+                        self.add_validation_error("ElectricUtility", "outage_durations",
+                                                f"Value is greater than the max allowable ({max_ts} - {max_start_time_step_input})")
```

Comment:
One comment that I meant to add in the PR review: since this only includes the "ElectricUtility" model, should this be in that model's `clean()` method in `models.py`?

Reply:
No because it uses max_ts, which is calculated in this function using Settings.time_steps_per_hour

Reply:
@hdunham thanks for clarifying, I missed that.  This is the right place then, thank you!